config/releases.exs

```diff
@@ -89,7 +89,9 @@ config :oli,
       ),
     favicons: System.get_env("BRANDING_FAVICONS_DIR", "/favicons")
   ],
-  node_js_pool_size: String.to_integer(System.get_env("NODE_JS_POOL_SIZE", "2"))
+  node_js_pool_size: String.to_integer(System.get_env("NODE_JS_POOL_SIZE", "2")),
+  screen_idle_timeout_in_seconds:
+    String.to_integer(System.get_env("SCREEN_IDLE_TIMEOUT_IN_SECONDS", "1,800,000"))
```

Comment:
I think it was an either or, this would imply that it's 1.8 million seconds. instead this would be in milliseconds. the rename to "IN_SECONDS" would mean that you leave it at `30*60` (for 30 minutes); might even make more sense to rename it to "IN_MINUTES" since it eases the configuration and it's not likely that anyone would configure this for less than 1 minute anyway